src/Nancy.Serialization.ServiceStack/project.json

```diff
@@ -18,9 +18,14 @@
     "Nancy": {
       "target": "project"
     },
-    "ServiceStack.Text": "3.9.71"
+    "ServiceStack.Text": "4.5.4"

```

Comment:
I think we can use `ServiceStack.Text.Core` for the netstandard target and get rid of the imports :smile: